proto/ddsketch_full.proto

```diff
@@ -0,0 +1,63 @@
+// Copied from https://github.com/DataDog/sketches-go/blob/master/ddsketch/pb/ddsketch.proto
```

Comment:
I think it would be a good idea to pin this link to a specific commit, in this case `8a1961cf57f80fbbe26e7283464fcc01ebf17d5c` instead of `master`